### PR TITLE
New package: mmark-2.0.38

### PR DESCRIPTION
**Pull request by bobertlo**

### CODE REVIEW

**Comment by bobertlo** [comment]
#8814

File: srcpkgs/mmark/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'mmark'
+pkgname=mmark
+version=2.0.38
+revision=1
+build_style=go
+go_import_path="github.com/mmarkdown/mmark"
+hostmakedepends="git"
+short_desc="Powerful markdown processor geared towards the IETF"
+maintainer="Robert Lowry <bobertlo@gmail.com>"
+license="2-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/mmark/template]: BSD-2-Clause

File: srcpkgs/mmark/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'mmark'
+pkgname=mmark
+version=2.0.38
+revision=1
+build_style=go
+go_import_path="github.com/mmarkdown/mmark"
```
Comment (maxice8) [inline srcpkgs/mmark/template]: not required to be quoted

**Comment by maxice8** [comment]
please add

closes #8814 

to the commit so it is closed when this PR is merged, regardless of the method.

**Comment by bobertlo** [comment]
@maxice8 thanks, fixed.

**Pull request merged**